climada/hazard/storm_europe.py

```diff
@@ -81,13 +82,35 @@ class StormEurope(Hazard):
     vars_opt = Hazard.vars_opt.union({'ssi_wisc', 'ssi', 'ssi_full_area'})
     """Name of the variables that aren't need to compute the impact."""
 
-    def __init__(self):
-        """Calls the Hazard init dunder. Sets unit to 'm/s'."""
-        Hazard.__init__(self, HAZ_TYPE)
-        self.units = 'm/s'
-        self.ssi = np.array([], float)
-        self.ssi_wisc = np.array([], float)
-        self.ssi_full_area = np.array([], float)
+    def __init__(self,
+                 units: str = 'm/s',
+                 ssi: Optional[np.ndarray] = None,
+                 ssi_wisc: Optional[np.ndarray] = None,
+                 ssi_full_area: Optional[np.ndarray] = None,
+                 **kwargs):
+        """Initialize a StormEurope object
+
+        Parameters
+        ----------
+        units : str
+            The units of the Hazard intensity. Defaults to 'm/s'
+        ssi : numpy.ndarray
+            The Storm Severity Index (SSI). 1d vector, same length as number of
+            storms. Defaults to an empty array.
+        ssi_wisc : numpy.ndarray
+            The Storm Severity Index (SSI) as reconded in the footprint files.
+            1d vector, same length as number of storms. Defaults to an empty
+            array.
+        ssi_full_area : numpy.ndarray
+            FIXME need to recover rationale for this attribute; no substantive
```

Comment:
@ThomasRoosli do you recall why we had this additional ssi_full_area thing in here? It's only ever used when computing a synthetic event set. I seem to remember that there was a good reason to not set the synthetic SSI to the same attribute name present in the reanalysis/WISC synthetic data. However, I don't remember, and we seem not to have documented it. I propose to simply drop this additional parameter. I would say this PR is as good an occasion as any.

Reply:
I would have to look closer into the code. From my memory: `ssi_full_area` is needed to conserve the index that is derived from the area and windspeed of the full geographic extend of a storm system. This is importent if a StormEurope object is cropped to a smaller region (e.g. a country).

Reply:
@ThomasRoosli: thanks for chipping in! In this case I'd be fine to keep the attribute and write just that into the description:
```
   Used to conserve the the index that is derived from the area and windspeed 
   of the full geographic extend of a storm system. Needed for cropping the 
   `StormEurope` object into a smaller region`
```